samples/python/multiview_calibration.py

```diff
@@ -0,0 +1,410 @@
+import sys, cv2 as cv, numpy as np, os, pickle, json, argparse, matplotlib.pyplot as plt, time, joblib, multiprocessing
```

Comment:
It is better to have 1 dependency per line.

Reply:
fixed